src/pymanopt/manifolds/positive_definite.py

```diff
@@ -113,20 +91,221 @@ def exp(self, point, tangent_vector):
+    """Manifold of symmetric positive definite matrices.
+
+    Points on the manifold and tangent vectors are represented as arrays of
+    shape ``k x n x n`` if ``k > 1``, and ``n x n`` if ``k == 1``.
+
+    Args:
+        n: The size of matrices in the manifold, i.e., the number of rows and
+            columns of each element.
+        k: The number of elements in the product geometry.
+
+    Note:
+        The geometry is based on the discussion in chapter 6 of [Bha2007]_.
+        Also see [SH2015]_ for more details.
+
+        The second-order retraction is taken from [JVV2012]_.
+    """
+
+    def __init__(self, n: int, *, k: int = 1):
+        self._n = n
+        self._k = k
+
+        if k == 1:
+            name = f"Manifold of positive definite {n}x{n} matrices"
+        else:
+            name = (
+                f"Product manifold of {k} positive definite {n}x{n} matrices"
+            )
+        dimension = int(k * n * (n + 1) / 2)
+        super().__init__(name, dimension)
+
+
+class HermitianPositiveDefinite(_positive_definite):
+    """Manifold of hermitian positive definite matrices.
+
+    Points on the manifold and tangent vectors are represented as arrays of
+    shape ``k x n x n`` if ``k > 1``, and ``n x n`` if ``k == 1``.
+
+    Args:
+        n: The size of matrices in the manifold, i.e., the number of rows and
+            columns of each element.
+        k: The number of elements in the product geometry.
+    """
+
+    def __init__(self, n: int, *, k: int = 1):
+        self._n = n
+        self._k = k
+
+        if k == 1:
+            name = (
+                f"Manifold of Hermitian positive definite ({n} x {n}) matrices"
+            )
+        else:
+            name = f"Product manifold of {k} ({n} x {n}) Hermitian positive definite"
+        dimension = int(k * n * (n + 1))
+        super().__init__(name, dimension)
+
+    def random_point(self):
+        # Generate eigenvalues between 1 and 2.
+        d = 1.0 + np.random.uniform(size=(self._k, self._n, 1))
+
+        # Generate an orthogonal matrix.
+        q, _ = multiqr(
+            np.random.normal(size=(self._n, self._n))
+            + 1j * np.random.normal(size=(self._n, self._n))
+        )
+        point = q @ (d * multihconj(q))
+        if self._k == 1:
+            return point[0]
+        return point
+
+    def random_tangent_vector(self, point):
+        k = self._k
+        n = self._n
+        if k == 1:
+            tangent_vector = multiherm(
+                np.random.randn(n, n) + 1j * np.random.randn(n, n)
+            )
+        else:
+            tangent_vector = multiherm(
+                np.random.randn(k, n, n) + 1j * np.random.randn(k, n, n)
+            )
+        return tangent_vector / self.norm(point, tangent_vector)
+
+    def zero_vector(self, point):
+        k = self._k
+        n = self._n
+        if k == 1:
+            return np.zeros((n, n), dtype=complex)
+        return np.zeros((k, n, n), dtype=complex)
```

Comment:
This I think we can also move to `_PositiveDefiniteBase` if we replace it with this:
```suggestion
    def zero_vector(self, point):
        k = self._k
        n = self._n
        if k == 1:
            return np.zeros((n, n), dtype=point.dtype)
        return np.zeros((k, n, n), dtype=point.dtype)
```